setup.cfg

```
@@ -22,9 +22,68 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+[metadata]
+name = invenio-webhooks
+version = attr: invenio_webhooks.__version__
+description = "Invenio webhooks module to create REST APIs."
+long_description = file: README.rst, CHANGES.rst
+keywords = invenio webhooks
+license = GPLv2
+author = CERN
+author_email = info@inveniosoftware.org
+platforms = any
+url = https://github.com/inveniosoftware/invenio-webhooks
+classifiers =
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU General Public License v2 (GPLv2)
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Internet :: WWW/HTTP :: Dynamic Content
+    Topic :: Software Development :: Libraries :: Python Modules
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: Implementation :: CPython
+    Development Status :: 1 - Planning
 
-[aliases]
-test=pytest
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.7
+zip_safe = False
+install_requires =
+    invenio-i18n>=1.3.2
+    invenio-oauthclient>=2.0.1
+    invenio-oauth2server>=1.3.6
+
+[options.extras_require]
+tests =
+    pytest-black>=0.3.0,<0.3.10
+    invenio-app>=1.3.2
+    invenio-celery>=1.2.4
+    invenio-db[postgresql,mysql,versioning]>=1.0.14,<2.0
+    pytest-invenio>=1.4.11
```

```suggestion
    pytest-invenio>=1.4.13
```

to include docker-services-cli that's making GH actions fail